orc8r/protos/bootstrapper.proto

```diff
@@ -71,3 +73,114 @@ service Bootstrapper {
   // Returns signed certificate.
   rpc RequestSign (Response) returns (Certificate) {}
 }
+
+// --------------------------------------------------------------------------
+// Protos for CloudRegistration and Registration servicer
+// --------------------------------------------------------------------------
+/**
```

Comment:
I find this block comment to be a little under-useful -- I'd rather this info be in a sequence diagram in the docusaurus, as I think it would be easier to read there

I'm not opposed to putting this here, but I just think it won't get read as-is

Reply:
Ok, made a tix for documenting later :)) #10576